verify/README

```diff
@@ -2,77 +2,91 @@
+The mounts normally default to /opt/cockpit-verify-secrets and /opt/cockpit-verify-images
+on the host.
 
-    # echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
-    # echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
-    # ( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
+# Deploying on a host
 
-Setup a 'cockpit' user:
+Make sure docker and atomic are installed and running:
 
-    # groupadd -g 1111 -r cockpit && useradd -r -g cockpit -u 1111 cockpit
-    # mkdir -p /home/cockpit/.ssh /home/cockpit/.config /home/cockpit/verify
-    # chown -R cockpit:cockpit /home/cockpit
+    $ sudo systemctl enable docker
+    $ sudo atomic install cockpit/infra-verify
+    $ sudo systemctl start cockpit-verify
 
-Fill in the following files with valid credentials able to post logs to sink and
-update github status:
+You may want to customize things like the operating system to test or number of jobs:
 
-    /home/cockpit/.ssh/id_rsa
-    /home/cockpit/.ssh/id_rsa.pub
-    /home/cockpit/.config/github-token
+    $ sudo mkdir -p /etc/systemd/system/cockpit-verify.service.d
+    $ sudo sh -c 'printf "[Service]\nEnvironment=TEST_JOBS=8\n" > /etc/systemd/system/cockpit-verify.service.d/jobs.conf'
+    $ sudo sh -c 'printf "[Service]\nEnvironment=VERIFY_IMAGES=/mnt/nfs/share/images\n" > /etc/systemd/system/cockpit-verify.service.d/images.conf'
 
-Optionally you can add a local whitelist with github usernames whose pull requests are considered for testing.
-Entries in this file are added to the in-tree cockpit/test/github-whitelist:
+## Troubleshooting
 
-    /home/cockpit/.config/github-whitelist
+Some helpful commands:
 
-For cockpituous, /home/cockpit/.ssh/config might look like this:
+    # journalctl -fu cockpit-verify
+    # systemctl stop cockpit-verify
 
-    Host fedorapeople.org
-        IdentityFile ~/.ssh/id_rsa
-        User cockpit
+## Updates
+
+To update, just pull the new container and restart the cockpit-verify service.
+It will restart automatically when it finds a pause in the verification work.
 
-The verification will use git, which complains if not set up properly. Do this as the cockpit user:
+    # docker pull cockpit/infra-verify
+
+## Deploying on Openshift
+
+The verify machine can run on an Openshift cluster.
+
+Build the secrets you prepared above by running this in a git repository checkout
+of this git repo. This will create a cockpit-verify-secrets.json file
+
+    $ sudo make verify-secrets
+
+Create a service account for use by the verify machines.  Currently the privileged SCC
+is used for scheduling the verify pod. This is because of the requirement to access
+/dev/kvm. Further work is necessary to remove this requirement.
 
-    # git config --global user.email INSERT_EMAIL
-    # git config --global user.name INSERT_NAME
+    $ oc create -f verify/cockpit-verify-account.json
+    $ oc adm policy add-scc-to-user priviliged -z verifier
```

Comment:
`priviliged` typo.

I'd add `$ shred -u verify/cockpit-verify-account.json` here as it contains a copy of the user's ssh private key and github token.